extension/README.md

```diff
@@ -0,0 +1,85 @@
+# Extensions
+*Note* This documentation is still in progress. For any questions, please reach
+out in the [OpenTelemetry Gitter](https://gitter.im/open-telemetry/opentelemetry-service)
+or refer to the [issues page](https://github.com/open-telemetry/opentelemetry-service/issues).
+
+Supported service extensions (sorted alphabetically):
+- [Health Check](#health-check)
+- [Performance Profiler](#pprof)
+- [zPages](#zpages)
+
+## Ordering Extensions
+The order extensions are specified for the service is important as this is the
+order in which each extension will be started and the reverse order in which they
+will be shutdown. The ordering is determined in the `extensions` tag under the
+`service` tag in the configuration file, example:
+
+```yaml
+service:
+  # Extensions specified below are going to be loaded by the service in the
+  # order given below, and shutdown on reverse order.
+  extensions: [health-check, pprof, zpages]
+```
+
+## <a name="health-check"></a>Health Check
```

Comment:
You can also mention that only one instance can be created per process. Same for zpages and pprof.